pyogrio/raw.py

```diff
@@ -174,6 +177,12 @@ def write(
                 f"Could not infer driver from path: {path}; please specify driver explicitly"
             )
 
+    if promote_to_multitype is None:
+        if geometry_type.startswith("Multi") and driver in ["GPKG", "FlatGeobuf"]:
```

Comment:
```suggestion
        promote_to_multitype = geometry_type.startswith("Multi") and driver in ["GPKG", "FlatGeobuf"]
```

You may also want to consider putting this list of drivers into a global variable (set) just to make it easy to extend later.